utils/augmentation.py

```diff
@@ -26,7 +26,11 @@ def compose_transforms(params, dataset):
             lst_trans.append(RandomRotationTarget(limit=params['training']['augmentation']['rotate_limit'],
                                                   prob=params['training']['augmentation']['rotate_prob']))
 
-    lst_trans.append(ToTensorTarget())
+    if params['training']['normalization']['mean'] and params['training']['normalization']['std']:
```

Comment:
Would recommend the systematic use of get_key_def as a way to add robustness to our scripts. May not be necessary here, but just making sure it's a conscious decision. 
For example, get_key_def can save us bugs if value for specific parameter or the parameter itself is omitted in yaml. in that case, it assigns a default value (if added as parameter)